rules.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: page
+title: Community Rules
+permalink: /rules/
+---
+
+##ITC should be considered SFW (Safe for Work). As a rule of thumb, before you post anything, consider if you'd want a co-worker or your boss to see it.
+
+###The following is not allowed:
+
+- Any content which is not in keeping with the Code of Conduct.
+- Blatantly sexual content (pornography, nudity, etc.)
+- Misogyny, ableism, sexism racism, nationalist insults etc.
+- Personal abuse.
+- Links to anything which would break the rules if posted directly.
+- Posting any private phone numbers.
+- Advocating criminality or violence.
+- Trolling (A 'troll' is someone who posts controversial, inflammatory or off-topic messages with the primary intent of provoking other users. These users should be reported to admins immediately.)
+- Posting libellous material.
+- Txt spk (text speak).
```

Comment:
Ah ok. Remove it if you wish.

The reasoning was:

With the character limitations in place for twitter etc, it is sometimes useful to use text speak to get a point across. However Slack has no similar character restrictions, so there's no reason to use it. Non native-English speakers may find it hard to understand. Others just find it annoying. 

I've no major issue with removing it though.